apps/condo/domains/settings/schema/MobileFeatureConfig.test.js

```diff
@@ -264,6 +265,27 @@ describe('MobileFeatureConfig', () => {
                         message: 'Wrong phone number format',
                     })
             })
+
+            test('ONLY_SERVICE_PROVIDER_ORGANIZATION_TYPE_IS_ALLOWED', async () => {
+                const [organization] = await createTestOrganization(admin)
+
+                await expectToThrowGQLError(
+                    async () => await createTestMobileFeatureConfig(admin, organization, {
+                        ticketSubmittingWithoutPhoneIsDisabled: true,
```

Comment:
Without phone?) is it possible?

Reply:
During the design process the field name changed, it should be ticketSubmittingForServiceProviderOrgIsDisabled.

There is a commonPhone field. When disabling tickets from a mobile phone in a managing company, filling out commonPhone is required. If tickets are disabled by the service provider then the phone must be optional